packages/web/src/components/bench/surfaces/browser-bench-surface.css

```
@@ -0,0 +1,3 @@
+:root[data-bench-capture-privacy="true"] webview[data-browser-tab-id] {
```

Stylelint (17.14.0)

[error] Unknown type selector "webview" (selector-type-no-unknown)

(selector-type-no-unknown)